content/01.abstract.md

```diff
@@ -1,7 +1,7 @@
 ## Abstract {.page_break_before}
 
-Scientific literature reviews are ideal for open, collaborative writing.
-Allowing any interested individual to contribute as an author can strengthen a review, providing broad and fair coverage of the subject matter.
-However, the traditional multi-author writing process breaks down at scale.
-We present techniques for overcoming the challenges of open manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and manuscript building.
+Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
+However, traditional research and multi-author writing processes break down at scale.
```

Comment:
How do you think Manubot helps with the "research" part?

Reply:
I could see an argument for using GH issues to discuss relevant papers, like with Deep Review, but that's not really part of Manubot -- the program -- as such.

Reply:
I understood this in the broad sense. Projects that do "open, collaborative research" need a way to write their papers. Additionally, many of these projects can be ongoing and at the "paper" stage for a long time. Therefore Manubot doesn't need to be a complete solution for all open, collaborative research needs, but is an important part of the manuscript preparation stage. @agitter was that what you were thinking?

At a larger scale, where research applies to a whole field, the more teams using manubot, the more open and collaborative the field will be.

Reply:
@slochower my reasoning was along the lines of what @dhimmel said.  For open projects, e.g. https://think-lab.github.io/p/rephetio/#realtime-open-science-thinklab, "traditional" practices (for some definition of traditional) no longer work well.  Having a distributed forum for discussion and synchronizing data, code, and results is necessary.

Manubot does not directly address that, but it is a component of an open distributed research workflow.  Analysis may be coordinated in one repository and writing in another, but I believe that using Manubot for writing will help enable this type of collaborative research.  Contributors could be incentivized by knowing that their work would be fairly credited and they would have the opportunity to write the research product and be recognized as authors.

If that is not coming through clearly or is an overreach given Manubot's current functionality, we can consider alternative phrasing.

Reply:
@dhimmel @agitter Okay, I see your points. I guess I bristled at the strong language. Certainly there have been many large, multi-author projects that worked before Manubot. They are definitely not ideal. They usually suck in multiple ways. But it can be done. It's just inelegant and error-prone. Having said that, I'm not sure how I would rephrase things. Maybe instead of saying "break down at scale" we could say "become very challenging to manage at scale" or something like that. Thoughts?

Also, @agitter, you mentioned:
> Analysis may be coordinated in one repository and writing in another, 

This is a good point. I tried putting Manubot inside another repository once, to bundle code and the writing, and so I could use relative paths to figures so, e.g., `../figure.png` would always pull the latest `figure.png` saved by the code. But that required changing a bunch of little things, so it seems easiest to keep the writing and analysis in separate repositories. We can revisit this in a Manubot Issue if it is necessary.

Reply:
@slochower would it be better to remove "multi-author" and say

> However, traditional research and writing processes break down at scale.

That leaves "traditional" more open to interpretation.  If "traditional" means passing around a Word document with comments or LaTeX files, that does not scale well and is feasible but quite painful with many authors.

I don't want the abstract to be bombastic, but I do like the phrase "break down at scale" because I find it central to the need for Manubot.

Reply:
I don't think that is an improvement, "multi-author" is a key point. I'm okay to leaving this sentence as it is. I forgot that it was the abstract, and I do like short, declarative sentences in the abstract.

Reply:
Okay.  This isn't perfect, but I'll merge these changes.  Please feel welcome to rewrite this sentence in a new pull request if you come up with better phrasing.